### ISSUE TITLE
Sudo in build images

### ISSUE THREAD

**Issue by simlu** [lambci/docker-lambda#88]
Any reason why we can't have extended permissions in build images? We are trying to use the build images as base in circle, but can't install e.g. global npm dependencies without hosting our own docker images.

What would be involved to change this? Any reason it shouldn't be done?

**Comment by mhart** [comment]
@simlu can you explain what you mean? What are you trying to do that's not working?

**Comment by simlu** [comment]
@mhart Thanks for the quick reply.

We successfully run lambci docker images locally to test our lambda functions and deploy from local.

This is done by using a custom Dockerfile like so (shortened example):
```Dockerfile
FROM lambci/lambda:build-python3.6

[...other stuff happening here...]

COPY requirements.txt /user/requirements.txt
RUN pip install -r /user/requirements.txt
RUN rm /user/requirements.txt

ENTRYPOINT bash
```

However now we want to set up CI and on circleci we can only define an image and then run commands inside it. So we can not install our pip requirements since `pip install` causes permission problems:

**Error:**
```sh
error: could not create '/var/lang/lib/python3.6/site-packages/sqlalchemy': Permission denied
```

Ideally we would just start `lambci/lambda:build-python3.6` directly and just run `pip install` inside. I'm really not sure how other people are doing this.

**Comment by mhart** [comment]
Yeah, I'm not sure what the problem is – probably something with your CI setup?

The user on the build images is root by default, so there are no restricted permissions:
```console
$ docker run --rm lambci/lambda:build-python3.6 whoami
root
```

**Comment by simlu** [comment]
@mhart Ah, interesting. So this is probably something on our end then. Let me do some digging into it! Thank you again very much for being so responsive!
